### ISSUE TITLE
Black screen appear if u slide multiple times

### ISSUE THREAD

**Issue by htarwara6245** [John-Lluch/SWRevealViewController#435]
Please try to test the slide in and slide out multiple times with gesture
http://www.appcoda.com/sidebar-menu-swift/
I am now currently learning from this tutorial and sometime black screen appear.i wonder why?

**Comment by John-Lluch** [comment]
Hi, this is an Apple animation bug. Please try setting `toggleAnimationType` to `SWRevealToggleAnimationTypeEaseOut`

**Comment by thihaaung6245** [comment]
yeah,it solved with your help.Thanks.But,I found another bug which was
If i try to do infinite left gesture without releasing my hands even the left menu appear and bounces,that menu doesn't appear when i release like it dismiss.

**Comment by cbh2000** [comment]
Even if this is an Apple bug, is there a workaround other than changing the ease type? @John-Lluch could you share more details specifically? I'm interested in looking into this bug.

**Comment by cbh2000** [comment]
Duplicate of #323.
